Compile warning: Duplicate resource name "net_PropertyNotSupportedException" is not allowed

Occurred during test runs of https://github.com/dotnet/corefx/pull/18170 for portablelinux_debug

@davidsh - duplicate resource added 4/10/2017 in https://github.com/dotnet/corefx/commit/bbe86c8f0e113d70d0d97936cc254889afbbe149#diff-bac3839a816a936d5db4ecb32728de17

````
./src/Resources/Strings.resx : warning MSB3568: Duplicate resource name "net_PropertyNotSupportedException" is not allowed, ignored. [/mnt/resource/j/workspace/dotnet_corefx/master/portablelinux_debug_prtest/src/System.Net.Requests/tests/System.Net.Requests.Tests.csproj]
````

--- a/src/System.Net.Requests/src/Resources/Strings.resx
+++ b/src/System.Net.Requests/src/Resources/Strings.resx
@@ -94,9 +94,6 @@
   <data name="net_PropertyNotImplementedException" xml:space="preserve">
     <value>This property is not implemented by this class.</value>
   </data>
-  <data name="net_PropertyNotSupportedException" xml:space="preserve">
-    <value>This property is not supported by this class.</value>
-  </data>
   <data name="net_nouploadonget" xml:space="preserve">
     <value>Cannot send a content-body with this verb-type.</value>
   </data>

